src/controls/selectors/ObjectTypeSelector.tsx

```diff
@@ -11,16 +11,23 @@ import { usePageParams } from '../PageParamsContext';
 const objectAmbiguousViews = [View.About, View.Details];
 
 const ObjectTypeSelector: React.FC = () => {
-  const { objectType, updatePageParams, view } = usePageParams();
-  const isCompact = useMediaQuery('(max-width: 1050px)');
+  const { objectType, replaceAllParams, view } = usePageParams();
+  const isCompact = useMediaQuery('(max-width: 1015px)');
```

Comment:
Can you restore this to maxWidth 1050px. Since you started on this the maxWidth for this layout was increased because we added a new object type (Censuses).
```suggestion
  const isCompact = useMediaQuery('(max-width: 1050px)');
```